scripts/full_check.sh

```diff
@@ -0,0 +1,111 @@
+#! /bin/bash
```

Comment:
```suggestion
#! /bin/bash -e
```

Reply:
It's usually better to use explicit `set -e` in case someone uses `bash script` to launch the script

Reply:
What about `set -e`? I think, it's very useful for error checking.